test/linproginterface.jl

```diff
@@ -82,29 +82,31 @@ function linprogsolvertest(solver::AbstractMathProgSolver, eps = Base.rtoldefaul
 
             test_result(m, v, c, MPB.Success, MPB.FeasiblePoint, MPB.FeasiblePoint, 1, [1.0, 0.0], [0.0, -1.0], [0.0, 0.0], [1.0], [1.0])
 
+        end
+    end
+end
+"""
             # add new variable to get:
             # max x + 2z
             # s.t. x + y + z <= 1
             # x,y,z >= 0
-            z = MPB.addvariable!(m, c, [1])
-            MPB.setattribute!(m, MPB.VariableLowerBound(), z, 0)
-            MPB.modifyobjective!(m, z, 2)
+            addvar!(m, [1], [1.0], 0, Inf, 2.0)
```

Comment:
Maybe it's an artifact of the diff, but it looks like some of the test changes are being reverted here. Is that intentional?